File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGLogLevel.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGLogLevel {
    YGLogLevelError,
    YGLogLevelWarn,
    YGLogLevelInfo,
    YGLogLevelDebug,
    YGLogLevelVerbose,
    YGLogLevelFatal;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGLogLevel forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGLogLevel.kt
package app.cash.redwood.yoga.internal.enums

enum class YGLogLevel {
    YGLogLevelError, YGLogLevelWarn, YGLogLevelInfo, YGLogLevelDebug, YGLogLevelVerbose, YGLogLevelFatal;

    fun getValue(): Int {
        return ordinal
    }

    companion object {

        fun forValue(value: Int): YGLogLevel {
            return values()[value]
        }
    }
}
